src/index.js

```diff
@@ -193,6 +210,12 @@ class HCaptcha extends React.Component {
 
       hcaptcha.reset(captchaId) // If hCaptcha runs into error, reset captcha - hCaptcha
       if (onError) onError(event);
+
+      // Reject promise if exists
+      if (typeof this.reject !== undefined) {
+        this.reject(new Error(event));
```

Comment:
Should we reject with the event or with a string?

Reply:
The event should be an instance of Error already, so I would do `this.reject(event)`.